src/rohan/WordsStringArray.java

```diff
@@ -0,0 +1,92 @@
+package rohan;
+
+import java.util.Scanner;
+
+/*Program 1: print all the words which does't have any digit. 
+String[] input = {"Masgj3Pw", "r4fsjk","techno","sfjk44lk"};
+output : techno
+
+Program 2: print all the words where digit sum is > 10. 
+String[] input = {"Ma1s3gj3Pw", "r4f3sjk","te3ch2no9","sfjk4p4plpk"};
+output : te3ch2no9
+*/
+
+public class WordsStringArray {
+	
+	public static void main(String[] args) {
+		String[] strArr;
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Program 1: Print all the words in string array which doesn't have any digit. Press - 1");
+		System.out.println("Program 2: Print all the words in string array where digit sum>10. Press - 2");
+		int input = sc.nextInt();
+		switch(input) {
+		case 1:
+			strArr = inputArray();
+			wordDoesntHaveDigit(strArr);
+			break;
+		case 2:
+			strArr = inputArray();
+			wordSumMoreThanTen(strArr);
+			break;
+		default:
+			System.out.println("Invalid Input");
+		}
+	}
+	
+	static String[] inputArray() {
+		Scanner sc = new Scanner(System.in);
```

Comment:
Make always parametrized methods, we never take input from service method, think, service method take input from it's method param